WindowsSandboxExtension/Constants.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WindowsSandboxExtension;
+
+internal sealed class Constants
+{
+    public const string ProviderId = "Microsoft.WindowsSandbox";
+    public const string ProviderDisplayName = "Windows Sandbox";
```

Comment:
Should this be localized?

Reply:
I do not think Windows Sandbox needs to be localized. See `HyperVExtension\src\HyperVExtension\Helpers\HyperVStrings.cs `